.gitignore

```diff
@@ -180,3 +180,6 @@ project.lock.json
 .vs/
 Tests/
 TestFiles/
+DocumentFormat.OpenXml.Tests/Spreadsheet.xltx
```

Comment:
It would be better to put these in the TestFiles directory instead and then they automatically get ignored